site/_docs/windows.md

```diff
@@ -16,6 +16,8 @@ A quick way to install Jekyll is to follow the [installation instructions by Dav
  2. Install Ruby via Chocolatey: `choco install ruby -y`
  3. Reopen a command prompt and install Jekyll: `gem install jekyll`
 
+Note: if step 3 fails due to an SSL error, save the file [cacert.perm](https://curl.haxx.se/ca/cacert.pem) into your rubygems folder "C:\tools\rubyXX\lib\ruby\X.X.X\rubygems\ssl_certs" and retry step 3.
```

Comment:
😱 I'm not going to trust a random certificate from `haxx.se` :-1:

There must be a better way to solve this problem.